tests/models/neural_networks/test_ealstm.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pickle
+import pytest
+
+from src.models.neural_networks.ealstm import EALSTM
+from src.models import EARecurrentNetwork
+
+from tests.utils import _make_dataset
+
+
+class TestEARecurrentNetwork:
+
+    def test_save(self, tmp_path, monkeypatch):
+
+        features_per_month = 5
+        dense_features = [10]
+        hidden_size = 128
+        rnn_dropout = 0.25
+        include_latlons = True
+        include_pred_month = True
+        include_yearly_means = True
+        yearly_mean_size = 3
```

Comment:
Sorry can you remind me why this has a size? Surely the yearly mean is a product of the number of `features_per_month` ?

Reply:
Yes, but its not trivial to work out what the relationship is (for instance, if there are monthly aggregates or surrounding pixels). So its easier to pass this as an argument to the model directly